### ISSUE TITLE
"print from close_date" causes problems on account open

### ISSUE THREAD

**Issue by blais** [beancount/beancount#10]
**[Original report](https://bitbucket.org/blais/beancount/issue/10) by Matthew Harris (Bitbucket: [matthew_harris](https://bitbucket.org/matthew_harris), GitHub: [mharris312](https://github.com/mharris312)).**

----------------------------------------

I'm not sure what all is going on here, but I'm sure it'll be pretty obvious to you.

```
#!

$ cat test.beancount
2000-01-01 open Assets:Checking
$ bean-query test.beancount 'print from close_date("XXX");'
Traceback (most recent call last):
  File "/Users/mharris/homebrew/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/cmd.py", line 214, in onecmd
    func = getattr(self, 'do_' + cmd)
AttributeError: 'BQLShell' object has no attribute 'do_print'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/shell.py", line 223, in default
    self.dispatch(statement)
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/shell.py", line 213, in dispatch
    return method(statement)
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/shell.py", line 284, in on_Print
    query_execute.execute_print(c_print, self.entries, self.options_map, file)
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/query_execute.py", line 73, in execute_print
    entries = filter_entries(c_print.c_from, entries, options_map)
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/query_execute.py", line 57, in filter_entries
    for entry in entries
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/query_execute.py", line 58, in <listcomp>
    if c_expr(entry)]
  File "/Users/mharris/homebrew/lib/python3.4/site-packages/beancount/query/query_env.py", line 156, in __call__
    close_entry, close_entry = context.open_close_map[args[0]]
AttributeError: 'Open' object has no attribute 'open_close_map'

```

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

This is indeed a bug. Will handle.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #10: "print from close_date" causes problems on account open.
